data.sql

```diff
@@ -88,6 +88,54 @@ WHERE name IN ('Angemon', 'Boarmon');
 
 COMMIT;
 
+-- INSERT VETS DATA
+BEGIN;
+
+ALTER SEQUENCE vets_id_seq RESTART WITH 1;
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Jack Harkness', 38, '2008-06-06');
+
+COMMIT;
+
+-- INSERT SPECIALIZATION DATA
+BEGIN;
+
+INSERT INTO specializations(vet_id, species_id) VALUES(1, 1);
+INSERT INTO specializations(vet_id, species_id) VALUES(3, 1);
+INSERT INTO specializations(vet_id, species_id) VALUES(3, 2);
+INSERT INTO specializations(vet_id, species_id) VALUES(4, 2);
+
+COMMIT;
+
+-- INSERT DATA IN VISITS TABLE
+ROLLBACK;
```

Comment:
- [ ] Please avoid rolling back these changes as the instructions does not say you should